bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/contentassist/AdditionalInfoController.java

```diff
@@ -52,7 +52,9 @@ class AdditionalInfoController extends AbstractInformationControlManager {
 	 * @since 3.2
 	 */
 	private static abstract class Timer {
-		private static final int DELAY_UNTIL_JOB_IS_SCHEDULED= 50;
+		/** delay until additional information is calculated **/
+		private static final int DELAY_UNTIL_JOB_IS_SCHEDULED= 0;
```

Comment:
As mentioned on more general comments, I think this value would better be removed all together and only OpenStrategy.getPostSelectionDelay() be used.

Reply:
OK.